### ISSUE TITLE
Transposition Shortcut

### ISSUE THREAD

**Issue by Fourniture** [musescore/MuseScore#24861]
### Your idea

A right-click-menu option that allows for easy transposing. Visualization:
![image](https://github.com/user-attachments/assets/9a410c3f-de33-4012-a857-7b3f88f1478b)

### Problem to be solved

It is cumbersome to go to "Tools" to transpose a passage, if you transpose frequently. When arrangers and orchestrators want one instrument double another an octave above or below, this would be a helpful option. Just select the passage, copy, paste, and then right click to transpose.

### Additional context

I am sure that a shortcut could be created by the user for this feature, but I expect a program to have what I need. This is a common operation which Musescore should have readily available.

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by SteveBlower** [comment]
Are you aware of the existing shortcuts CTRL+Up Arrow to transpose up an octave and CTRL+Down arrow to transpose down an octave?

**Comment by Fourniture** [comment]
No, I was not. Thank you for mentioning them. However, I would still suggest that this appear as a feature, as the right-click menu seems to be the first place to look if one wanted something transposed.

**Comment by SteveBlower** [comment]
Are you also aware of the other keyboard shortcuts for transposition:

Arrow up/down - transpose by a semitone 

ALT+SHIFT+ up/down - move notes diatonically.

See https://musescore.org/en/handbook/4/transposition#Transposing_with_keyboard_shortcuts

**Comment by FrancRos31** [comment]
Maybe is it better to move the "Transpose..." dialog from "Tools" to the right-click menu?

**Comment by Fourniture** [comment]
I personally expect to find "Transpose" in "Tools"; it's just that it's the second place I would look. There's nothing wrong with having two access paths; "Staff/Part Properties" is found both in the right-click menu and the properties sidebar. Notice, however, that I recommended that only octave transposition is found in the right-click menu (and a "more..."). The whole purpose of the right-click shortcut is to avoid having to deal with the large transposing menu every time you need to transpose. It would reduce the process from 6 clicks (Tools -> Transpose -> By Interval -> Up/Down -> Perfect Octave -> OK) to only two.

**Comment by SteveBlower** [comment]
But as per my previous comment, octave transpositions already have default keyboard shortcuts.  Do we really need **three** ways to do the same thing?

**Comment by Fourniture** [comment]
Consider a text writing program. You can get "Properties" by clicking a button, or in the right-click menu, or in the top toolbar. You can also get subscript or superscript in any of those three options. Those three options also exist for changing the font! So clearly there is precedent for three options. In this case, is it unnecessary duplication? I think not, because the two current ways to transpose something are not immediately evident. I repeat; I expect to find features that transform notes, text, images, etc. in the right-click menu. It's a standard workflow.

**Comment by FrancRos31** [comment]
> I personally expect to find "Transpose" in "Tools"; it's just that it's the second place I would look. There's nothing wrong with having two access paths; "Staff/Part Properties" is found both in the right-click menu and the properties sidebar. Notice, however, that I recommended that only octave transposition is found in the right-click menu (and a "more..."). The whole purpose of the right-click shortcut is to avoid having to deal with the large transposing menu every time you need to transpose. It would reduce the process from 6 clicks (Tools -> Transpose -> By Interval -> Up/Down -> Perfect Octave -> OK) to only two.

Having the menu in two places seems a good idea, let's see what the devs think about it.
